<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_new_obj</class>
 <widget class="QDialog" name="fm_new_obj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый объект</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип объекта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_type">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Дом</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Квартира</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Дача</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Кол-во комнат</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sb_rooms">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Площадь (м.кв)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="sb_square">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Цена ($)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sb_price">
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Город</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ed_city"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_add">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
